packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md

```diff
@@ -59,9 +59,16 @@ Create an OAuth 2.0 service app integration using the Admin Console.
 
 ### Assign admin roles to the OAuth 2.0 service app
 
-> **Note:** These instructions are for Production orgs with the **Assign admin roles to public client apps** feature enabled as well as Preview orgs.<br><br>
+> **Note:** Use these instructions for
+> * Preview orgs <br>
+>   If you want to automatically assign the super admin role to all newly created custom API service apps, you can enable the **Public client app admins** org setting. See [Assign admin roles to apps](https://help.okta.com/okta_help.htm?type=oie&id=csh-work-with-admin-assign-admin-role-to-apps).
```

Comment:
I feel that "newly created custom API service apps" was confusing - did that imply service apps recently created? or any service app that they create after they enable the feature? Please LMK if you have questions.

```suggestion
>   To automatically assign the super admin role to all custom API service apps that you create, enable the **Public client app admins** org setting. See [Assign admin roles to apps](https://help.okta.com/okta_help.htm?type=oie&id=csh-work-with-admin-assign-admin-role-to-apps).
```